Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-settings/test-launch-dialog.html

```diff
@@ -7,10 +7,10 @@ <h1 class="pull-left" md-dialog-title>Share Test</h1>
     <div class="form-group">
       <label>Global URL</label>
       <div class="url-container">
-        <input type="text" class="form-control" [(ngModel)]="testLink" name="Test Link" />
-        <em class="material-icons" ngxClipboard [cbContent]="testLink" (cbOnSuccess)="copiedContent = true">content_copy</em>
+        <input type="text" class="form-control" [(ngModel)]="testLink" name="TestLink" id="TestLink" #TestLink onblur="this.select();" />
```

Comment:
Yes, I am changing the name used for "id" and "name" from "TestLink" to "Link" or shall I change the name of the variable from "testLink" to something else?